python/thunder/utils/ec2.py

```diff
@@ -23,7 +23,6 @@
 #
```

Comment:
yeah. (1) I switched to os.execv() rather than os.system() so I could just pass the list of args, but (2) os.execv() was having trouble calling "python". then I realized that all thunder-ec2 did was add to the python path then call utils/ec2.py, so I figured it was cleaner to just point the `thunder-ec2` entry point to utils/ec2.py rather